content/en/blog/2022/K8s Triage bot Helper CI Job_ ‘make update’.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: blog
+title: "K8s Triage bot Helper CI Job: ‘*make update*’"
+date: 2022-03-15
+slug: k8s-triage-bot-helper-ci-job
+---
+
+# K8s Triage bot Helper CI Job: ‘*make update*’
+
```

Comment:
Add an author attribution here. Also, you don't need a heading “Introduction”; that can go without saying.